src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/CreateInvitation/CreateInvitationCommandHandlerTests.cs

```diff
@@ -19,6 +25,16 @@ public class CreateInvitationCommandHandlerTests
         private Mock<IFusionMeetingClient> _meetingClientMock;
         private Mock<IInvitationRepository> _invitationRepositoryMock;
         private Mock<IUnitOfWork> _unitOfWorkMock;
+        private Mock<ICommPkgApiService> _commPkgApiServiceMock;
+        private Mock<IMcPkgApiService> _mcPkgApiServiceMock;
+        private Mock<IPersonApiService> _personApiServiceMock;
+        private Mock<IFunctionalRoleApiService> _functionalRoleApiServiceMock;
+
+        private const string FrCode = "FR1";
```

Comment:
FunctionalRoleCode for readability. Also global variables should use convention _camelCase